rosbag2_transport/include/rosbag2_transport/play_options.hpp

```diff
@@ -53,6 +54,9 @@ struct PlayOptions
   // Start paused.
   bool start_paused = false;
 
+  // Start time from which the player should play the bag.
+  rcutils_time_point_value_t starting_time = INT64_C(0);
```

Comment:
nitpick: I would prefer to call this `start_time`. Or maybe even `start_offset`

The docstring should definitely be specific that this is an offset. "Time to start  playback as an offset from the beginning of the bag"

Reply:
To be honest, `start_time` does not do justice to the difference in meaning between this field and `starting_time` from `BagMetadata::starting_time`. I'm inclined to call it something like `start_offset` or `start_from` instead - probably the former.

I'll take extra care to note its nature in documentation.

Reply:
Renamed in https://github.com/ros2/rosbag2/pull/931/commits/ec56a7a01c7ec726f9f3744ceaff410bd14171a4.